### DISCUSSION TITLE
Wix v4 - Error WIX0400	The Condition element contains illegal inner text: 'Installed OR (VersionNT >= v6.3)'

### DISCUSSION THREAD

**Discussion by garevaul** [wixtoolset/issues#7344] [Questions] [answered]
I used HeatWave and Updated my Wix v3 project to Wix v4, but I am getting these errors for this Code;

I checked this Question but I still cannot figure it out.
https://github.com/orgs/wixtoolset/discussions/7197

```
<bal:Condition Message="This Application requires Windows 10 or higher to run.">
	<![CDATA[Installed OR (VersionNT >= v6.3)]]>
</bal:Condition>
```

Error	WIX0400	The Condition element contains illegal inner text: 'Installed OR (VersionNT >= v6.3)'.
Error	WIX0020	The Condition element's inner text cannot be an empty string or completely whitespace. If you don't want a condition, then simply remove the entire Condition element.

AND Similarly

```
<bal:Condition Message="Windows 10 Creators update or later is required.">
	<![CDATA[Installed OR (CBNumber >= 1703)]]>
</bal:Condition>
```
Error	WIX0400	The Condition element contains illegal inner text: 'Installed OR (CBNumber >= 1703)'.
Error	WIX0020	The Condition element's inner text cannot be an empty string or completely whitespace. If you don't want a condition, then simply remove the entire Condition element.

Regards

**Comment by barnson** [comment, marked as answer]
Move the inner text without the CDATA into a `Condition` attribute. (The converter missed that change.)

**Reply by garevaul** [reply]
Thanx @barnson

Worked excellent.

Regards @garevaul
